modin/pandas/test/test_dataframe.py

```diff
@@ -2358,13 +2358,13 @@ def test_interpolate(self):
 
     def test_kurt(self):
```

Comment:
We should be testing parameters here as well to ensure that we have the right behaviors.